operator/integration-tests/model-in-image/mii-sample-wrapper/build-model-image.sh

```diff
@@ -72,9 +72,7 @@ function output_dryrun() {
 MODEL_YAML_FILES="$(ls $WORKDIR/$MODEL_DIR/*.yaml | xargs | sed 's/ /,/g')"
 MODEL_ARCHIVE_FILES=$WORKDIR/$MODEL_DIR/archive.zip
 MODEL_VARIABLE_FILES="$(ls $WORKDIR/$MODEL_DIR/*.properties | xargs | sed 's/ /,/g')"
-if [ "$WDT_DOMAIN_TYPE" = "WLS" ]; then
-  CHOWN_ROOT="--chown oracle:root"
-fi
+CHOWN_ROOT="--chown oracle:root"
```

Comment:
If this is necessary, then all doc references to same requirement need update as well.

Use the following to find locations (from the top of the source tree):

```
 find . -name "*.md" | xargs grep "chown oracle:root"
```

Reply:
Sure. Thanks Tom

Reply:
Is this really necessary? Looping in @ddsharpe.

Reply:
We presently have doc like the following:

```
{{% notice note %}}
If you are taking the `JRF` path through the sample, then remove `--chown oracle:root` from the `imagetool.sh` command below. 
{{% /notice %}}
```

Reply:
Notes: The documentations we would need to change:
weblogic-kubernetes-operator/blob/develop/documentation/3.0/content/samples/simple/domains/model-in-image/initial.md

oracle/weblogic-kubernetes-operator/blob/develop/documentation/3.0/content/samples/simple/domains/model-in-image/update3.md

Same for documentation release 3.1 and 3.2

Reply:
The `develop` branch is closed. Please look at the `main` branch. The "documentation/3.0" directory is archived documentation from the 3.0 release. This pattern is similar for the other numbered directories. In particular, the "documentation/3.2" directory is the current live documentation. The "documentation/staging" directory is the pre-release documentation for our next minor release. 

Since the FMW images have changed, it probably makes sense to fix this issue every where that you find it.

Reply:
Just filed doc JIRA: https://jira.oraclecorp.com/jira/browse/OWLS-88635
I will use this JIRA to track needed doc change.

Reply:
For the doc change, we need to remove the below sentence since right now both FMW and WLS mii build need --chown oracle:root
{{% notice note %}}
If you are taking the `JRF` path through the sample, then remove `--chown oracle:root` from the `imagetool.sh` command below. 
{{% /notice %}}

Reply:
@rjeberhard Oh yeah... why I was still looking at develop branch :(
documentation/3.2 and documentation/staging are the branches I need to change, right?

Reply:
I suspect there are other locations in the doc for image tool command line - those would need to be checked as well to make sure that they include the chown.